pages/wp-content.jsx

```diff
@@ -0,0 +1,15 @@
+var React = require('react');
+var Page = require('../components/page.jsx');
+var WpContentLoader = require('react-wp-content-loader');
+
+// [TODO] 404 needs to be handled properly. Now we just make all the 
+//        non-component pages as wp-pages
+var WpContent = React.createClass({
+  render: function() {
+    return (
+      <WpContentLoader wpUrl="teachmozillaorg.wordpress.com" wpPostSlug={this.props.params.wpSlug} />
```

Comment:
as a [stateless component](https://toddmotto.com/stateless-react-components/), we can probably rewrite this:
```
var WpContent = function(props) { return <WpContentLoader wpPostSlug={props.params.wpSlug} ... />; };
```